py/desisim/survey_release.py

```diff
@@ -6,103 +6,89 @@
 
 """
 from __future__ import division, print_function
-from pkg_resources import resource_filename
 import os
 import desisim
 import numpy as np
 import healpy as hp
 import fitsio
-from astropy.io import fits
 from astropy.table import Table, vstack
 from desiutil.log import get_logger
 from scipy.stats import rv_discrete
 from desitarget.targetmask import desi_mask
-import desimodel.footprint
+from desimodel.io import load_tiles
+from desimodel.footprint import tiles2pix, is_point_in_desi, radec2pix
```

Comment:
Small readability comment: You can import `load_tiles` as `desimodel_load_tiles` so that it is easier to understand in the code below. Not a big deal.